src/script/event/EventRepository.ts

```diff
@@ -324,6 +324,9 @@ export class EventRepository {
       case EVENT_TYPE.CALL:
         amplify.publish(WebAppEvents.CALL.EVENT_FROM_BACKEND, event, source);
         break;
+      case EVENT_TYPE.FEDERATION:
+        amplify.publish(WebAppEvents.FEDERATION.EVENT_FROM_BACKEND, event, source);
+        break;
       case EVENT_TYPE.CONVERSATION:
         amplify.publish(WebAppEvents.CONVERSATION.EVENT_FROM_BACKEND, event, source);
         break;
```

Comment:
note for the future, I'd like the EventRepo to not really filter the type of events. 
It should just send all the events in a single event and all the repos subscribing to it will make the work of filtering depending on the event type. :)